src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/HeapSizeIncreaseAction.java

```diff
@@ -37,7 +44,17 @@
   public HeapSizeIncreaseAction(@Nonnull final AppContext appContext) {
     super(appContext);
     this.esNode = new NodeKey(appContext.getMyInstanceDetails());
-    this.canUpdate = Runtime.getRuntime().totalMemory() > 200 * GB_TO_B;
+    this.canUpdate = Runtime.getRuntime().totalMemory() > 200;
```

Comment:
Needs * GB_TO_B on the RHS

Reply:
Yikes! I updated this. Looks like one of my commits was missing. Added it now.